splash/render_options.py

```diff
@@ -20,14 +20,15 @@ def __init__(self, data):
         self.data = data
 
     @classmethod
-    def fromrequest(cls, request):
+    def fromrequest(cls, request, max_timeout=defaults.MAX_TIMEOUT):
         """
         Initialize options from a Twisted Request.
         """
 
-        # 1. GET / POST data
         data = {key: values[0] for key, values in request.args.items()}
+        data.setdefault('max-timeout', max_timeout)
 
+        # 1. GET / POST data
```

Comment:
"GET/POST data" comment is for `request.args.items`, so it makes more sense to put it above `data = {key: values[0] for key, values in request.args.items()}`